src/main.cpp

```diff
@@ -2239,16 +2239,23 @@ CAmount GetBlockSubsidy(int nHeight, const Consensus::Params& consensusParams)
     }
 }
 
+static std::atomic<bool> IBDLatchToFalse{false};
```

Comment:
Renamed `latchToFalse` to `IBDLatchToFalse` because that variable had to move outside of the scope of `IsInitialBlockDownload()`, so I wanted the name to have less chance of conflicting.